EIPS/eip-7044.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 7044
+title: Perpetually Valid Signed Voluntary Exits
+description: Lock voluntary exit signature domain on capella for perpetual validity
+author: Lion (@dapplion)
+discussions-to: https://github.com/ethereum/consensus-specs/pull/3288
```

Comment:
you might need to open a discussion or ethereum-magicians.org or https://ethresear.ch

Reply:
You need to create a discussion thread on ethereum-magicians.org
See Danny's as an example:
https://ethereum-magicians.org/t/eip-7045-increase-max-attestation-inclusion-slot/14342